scripts/p2p-test-tool.py

```diff
@@ -130,7 +131,8 @@
 client = docker.from_env()
 RNODE_CMD = '/opt/docker/bin/rnode'
 # bonds_file = f'/tmp/bonds.{args.network}' alternate when dynamic bonds.txt creation/manpiulation file works
-bonds_file = os.path.dirname(os.path.realpath(__file__)) + '/demo-bonds.txt'
+src_bonds_file = os.path.dirname(os.path.realpath(__file__)) + '/demo-bonds.txt'
```

Comment:
`os.path.join()` is a better `+` for paths

Reply:
Running on Windows is no the only reason to use it.  It eliminates duplicated slashes which become a problem if the path needs to be processed further.  It also indicates to the reader that a string is indeed a path.

Reply:
`dirname()` returns path without trailing `/` (I think). Feel free to checkout, make changes and push into my branch.